### ISSUE TITLE
Rate limit error occured.

### ISSUE THREAD

**Issue by wan54** [contentful/contentful-import#27]
```
Rate limit error occured. Waiting for 9.438011796730115e+116 ms before retrying....
Rate limit error occured. Waiting for 9.985945945398726e+116 ms before retrying....
Rate limit error occured. Waiting for 9.284097710025449e+116 ms before retrying....
Rate limit error occured. Waiting for 9.75815309707582e+116 ms before retrying....
Rate limit error occured. Waiting for 9.290254273493636e+116 ms before retrying....
Rate limit error occured. Waiting for 1.0299930682276244e+117 ms before retrying....
```

We have ~900 entries and the import never finishes.

Like contentful-export, perhaps we want to allow the rate limit to be set by the user for the import?

**Comment by Khaledgarbaya** [comment]
Hi @wan54,
which version are you using ?

**Comment by wan54** [comment]
4.0.0

**Comment by Khaledgarbaya** [comment]
Let me check with the backend team and get back to you

**Comment by wan54** [comment]
Thanks. It's still throwing the same error and eventually the import halts

**Comment by Khaledgarbaya** [comment]
Hi @wan54,
can you create a support ticket and send us the space-id. Just mention the Github issue and they will direct you to me.
Best,
Khaled

**Comment by lwakefield** [comment]
I am having the same issue. This happens mid-way through the import. It seems like this is there is some sort of bug managing async because when it gets to this stage, it looks like it is making a very large amount of requests to contentful >20/s (eyeballing the logs)

**Comment by lwakefield** [comment]
Just as a follow-up, whether this was caused by the actual import, or was a result of the large amount of requests made afterward - I don't know.

BUT - It seems like I have hit the hourly api rate limit.This is particularly concerning considering it seems like api rates are not per space or user, so it has affected our entire organization. So in this case, attempting to run a backup has bricked our entire organization, hopefully for no more than an hour...

**Comment by Khaledgarbaya** [comment]
Hi @lwakefield,
1. The tool is using rate limit headers to determine when it should retry, but it seems like there is a backend issue. Would you please open a support ticket and tell us your spaceID, so the backend engineers can take a look.
2. The rate limit is per organization, so if you did hit the rate limit the entire org will be affected, even when using the web app.

**Comment by lwakefield** [comment]
Thanks @Khaledgarbaya, I have dropped a line to support.

I know this isn't directly related to `contentful-import` but is definitely of note for this issue. The space which I was running the tool against seems to have been removed. I do not believe anyone in the organization has removed the space, so I suspect it has been removed with respect to this issue? I will deal with this through the support channels, however, I think it is worth sharing here.
